### ISSUE TITLE
Equation wrapping or linebreak not working - overflow

### ISSUE THREAD

**Issue by uc-apa** [mathjax/MathJax#2140]
In the window or main container, the line breaks are working fine, but when the equation is inside any class having padding or less width than main window, the equation overflows. I tried with longer equations, in both outer window and inner window. And it seems that the container width is universal. It is not wrapping content as per its parent width.

Below is the screenshot of the issue and the mathjax config also

![image](https://user-images.githubusercontent.com/25792136/56452183-3e474300-634d-11e9-935d-698eba9eefab.png)
```javascript
<script src="MathJax/MathJax.js">
            MathJax.Hub.Config({
                extensions: ["tex2jax.js"],
                jax: ["input/TeX","output/SVG"],
                TeX: {extensions: ["AMSmath.js", "AMSsymbols.js"]},
                showMathMenu: false,
                showMathMenuMSIE: false,
                CommonHTML: { linebreaks: { automatic: true, width: "75% container" } },
                "HTML-CSS": { linebreaks: { automatic: true, width: "75% container" } },
                SVG: { linebreaks: { automatic: true, width: "75% container" } },
                tex2jax: {
                    inlineMath: [["\\\\begin{equation}","\\\\end{equation}"]],
                    processEnvironments: false
                },
                messageStyle: "none"
            });
        </script> 
```
[From docs - linebreaks](http://docs.mathjax.org/en/latest/options/output-processors/CommonHTML.html)

**Comment by dpvc** [comment]
I am not able to reproduce your result.  My test cases all break properly taking padding and container width into account.  Can you provide a minimal example in jsFiddle or codebin, or at least a link to a page the exhibits the problem?  There is really not enough information here to diagnose the problem.

**Comment by uc-apa** [comment]
Yes, I tried debugging more and found the steps to produce this issue.
If we add a tag which is display: none; and it has padding-left as well, then it will not be wrapped. After that when we make is visible, the user can clearly see the unwrapped equation. Here, both display:none and padding-left together cause the issue. Please have a look to the below jsFiddle.
[jsFiddle](http://jsfiddle.net/fuLhnprb/1/)

**Comment by dpvc** [comment]
MathJax can not measure the sizes of containers that have `display:none` set (as the browser does not lay out those elements, so their sizes are not computed).  Possible work-arounds for that are given in several places: [MathJax wiki](https://github.com/mathjax/mathjax-docs/wiki/Dealing-with-display:none), [this blog entry](https://www.peterkrautzberger.org/0165/), [this talk](http://www.math.union.edu/~dpvc/talks/2013-01-12.jmm/examples.html) (click the mouse or press space to reveal greyed out text).

**Comment by uc-apa** [comment]
I am able to fix the issue with MathJax.Hub.Queue(["Reprocess",MathJax.Hub,revealedDiv]);. Thank you for the support and quick response.
